packages/grid/x-data-grid/src/components/cell/GridActionsCell.tsx

```diff
@@ -104,10 +227,39 @@ GridActionsCell.propTypes = {
    * @deprecated Use the `apiRef` returned by `useGridApiContext` or `useGridApiRef` (only available in `@mui/x-data-grid-pro`)
    */
   api: PropTypes.any.isRequired,
+  /**
+   * The mode of the cell.
+   */
+  cellMode: PropTypes.oneOf(['edit', 'view']).isRequired,
   /**
    * The column of the row that the current cell belongs to.
    */
   colDef: PropTypes.object.isRequired,
+  /**
+   * The column field of the cell that triggered the event.
+   */
+  field: PropTypes.string.isRequired,
+  /**
+   * A ref pointing to the element that should receive focus.
```

Comment:
I do not see the ref passed to an element

```suggestion
   * A ref allowing to set imperative focus.
```

Reply:
Yeah, it's not passed to an element only on this type of column. But on others, it could be. `actions` has internal navigation so we need to handle it differently. Ideally `focusElementRef` would go to an element.

How about?

```suggestion
   * A ref allowing to set imperative focus.
   * It can be passed to the element that should receive focus.
```